apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -65,7 +62,7 @@ const DisplayServiceAccountCard = (props: Props) => {
         break;
       default:
         setUnknownError(error);
-        throw new Error('Unhandled error exception');
+        throw new Error(JSON.stringify(error));
```

Comment:
Hm, I have a slightly different idea about what we could/should do here. (But it would be interesting to see how Sentry handles both cases.)

I believe we should just do:

```js
throw error
```

Since all we're doing here is re-throwing the error we caught above, since we weren't able to handle it.

An alternative would be

```js
throw new Error('Something something something', { cause: err })
```

`cause` is a newish language feature (but well supported: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause) which preserves the original error but allows you to throw something new if you feel like it.

Honestly we have nothing to add in this case, so I strongly recommend going with `throw error`. 😄